### ISSUE TITLE
Task: create blog post with screencast that shows how to use the plugin

### ISSUE THREAD

**Issue by adietish** [redhat-developer/intellij-kubernetes#227]
once #223 made it to master we should create a writeup (and screencast) that shows how to edit resources in intellij-kubernetes

**Comment by adietish** [comment]
This is done by @mohitsuman.

**Comment by adietish** [comment]
@mohitsuman could you progress on this?

**Comment by mohitsuman** [comment]
I am going to take this up for the upcoming sprint and focus on the latest release workflow.

**Comment by adietish** [comment]
I'll write a blogpost for the screencast I created: https://youtu.be/mfdrMDbWk00

**Comment by adietish** [comment]
The blog post was submitted to Red Hat Developers. Awaiting approval.
I will recreate the screencast to match the blog post (has additional steps)

**Comment by adietish** [comment]
Update: the blog post had to be turned into a 3 part learning path. This is because they only accept a maximum of 6 images in a blog post. The 19 I have made a series of 3 posts that then form a so called "learning path".
I submitted it on 14th of August and I am awaiting their feedback.

ps. reported 3 weeks of waiting time before his was approved.

**Comment by adietish** [comment]
Update: Blog team came back to me asking for clarifications (24th of August). I submitted them on the same day and am waiting for them get back to me since then.

I talked to Max Andersen/Quarkus who agreed that the post could get posted on the Quarkus blog. It should be shortened and focused on the Quarkus aspects though. He promised to try the instructions and come back with suggestions (17th of August)
